cloudbuild.yaml

```diff
@@ -8,103 +8,13 @@ steps:
       - "-c"
       - |
         git init
-        git remote add origin https://github.com/getsentry/$REPO_NAME.git
+        git remote add origin https://github.com/armenzg/$REPO_NAME.git
         git fetch --depth=1 origin $COMMIT_SHA
         git reset --hard FETCH_HEAD
         git config -f .gitmodules submodule.core.url https://github.com/getsentry/$REPO_NAME.git
+        curl -u ":$_GH_TOKEN" \
```

Comment:
@BYK I managed to trigger the `integration.yml` workflow in my repo.
I had to add a `_GH_TOKEN` with a personal token with `repo` and `workflow` access
![image](https://user-images.githubusercontent.com/44410/127046702-390c0075-f6bc-4709-aa8f-23fd1741eb0d.png) to my GCloud  account.

I feel this is excessive scope just to trigger a workflow.

Tomorrow I will look into using Google Secret.

Docs: https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values

I think it's excessive access and nothing we can do about it; what do you think?

Reply:
>I think it's excessive access and nothing we can do about it; what do you think?

I agree and I agree. GitHub being GitHub :shrug:

Reply:
Are you still comfortable with me pursuing this approach?
It would be ideal if we could use GHA and use the Google runners.

Who will be the issuer of the personal token? One of the `relay` admins? A bot account? Someone who only has `admin` access to `relay`?

Reply:
I think we can use @getsentry-bot account as we use it for getsentry/publish with same purposes and almost exactly same scope (sans workflow) anyway.